src/matcha_ml/core/core.py

```diff
@@ -24,31 +24,49 @@ def get(
         Dict[str, Dict[str, str]]: the information of the provisioned resource.
 
     Raises:
+        MatchaError: Raised when the matcha state has not been initialized
         MatchaError: Raised when the matcha.state file does not exist
         MatchaInputError: Raised when the resource or property name does not exist in the matcha.state file
     """
     matcha_state_service = MatchaStateService()
+    remote_state = RemoteStateManager()
```

Comment:
Nit: Probably outside of the scope of this ticket, but should these classes be harmonized (service vs. manager). For this module, should we use remote_state_service or remote_state_manager as a variable name?

Reply:
I agree, we could create a ticket for this, harmonizing everywhere with same variable/class name.